frontend/website/docs/developers/archive-node.md

```diff
@@ -16,7 +16,7 @@ coda daemon -propose-key ~/keys/my_wallet -rest 3085 -archive-port 3086
 - Coda daemon pushes diff into Archive process
 - Archive process transforms diff into [GraphQL](https://graphql.org/) query
 - Archive process sends GraphQL query to [Hasura process](https://hasura.io/)
-- Hasura transforms GraphQL query into SQL write 
+- Hasura transforms GraphQL query into SQL write
```

Comment:
Why do we say SQL write vs SQL query?

Reply:
we can be more specific and say "GraphQL mutation"